client/lib/menu-data/test/test-menu-data.js

```diff
@@ -297,29 +297,6 @@ describe( 'MenuData', function() {
 		} );
 	} );
 
-	describe( 'saveMenu', function () {
```

Comment:
:) Could you please update this rather than ripping it out? wpcom is a bit awkward to stub due to it's chaining interface, so we tend to nock out the `public-api` responses. Ping me if you need help with the async stuff.